src/components/JoinSwapExternAmount.svelte

```diff
@@ -537,7 +547,7 @@
         frozeQuote = quote;
         modal.open();
       }}
-      disabled={error || isLoading || balanceError || amount.label <= 0}
+      disabled={error || isLoading || balanceError || sellAmount.label <= 0}
```

Comment:
sellAmount.label cannot be < 0 from line 126, so it should be only sellAmount.label === 0